docker/bionic/Dockerfile

```diff
@@ -21,6 +21,7 @@ FROM ubuntu:bionic AS build_deps
 LABEL maintainer="Apache NiFi <dev@nifi.apache.org>"
 
 ARG MINIFI_VERSION
+ARG DOCKER_MAKE_COMMAND=""
```

Comment:
This should be `DOCKER_MAKE_TARGET`, instead of `DOCKER_MAKE_COMMAND`.

Also, I would prefer to make the default make target `"all"`, instead of `""` implicitly defaulting to `"all"`.